formats/travel_advice_index/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+     "organisations": {
```

Comment:
`organisations` is included `base_links.json` (https://github.com/alphagov/govuk-content-schemas/pull/153), so we don't need it here anymore.